soap/soap_types.bal

```diff
@@ -19,23 +19,24 @@ public type SoapVersion "SOAP11"|"SOAP12";
 @final public SoapVersion SOAP11 = "SOAP11";
 @final public SoapVersion SOAP12 = "SOAP12";
 
-documentation {
-    F{{headers}} The array of SOAP headers for the SOAP envelop to send to the endpoint
-    F{{payload}} The xml of SOAP payload for the SOAP envelop to send to the endpoint
-    F{{soapVersion}} The version of SOAP
-    F{{soapAction}} The value of SOAP Action to send to the endpoint
-    F{{^"from"}} The value for the source endpoint parameter used for WS-Addressing
-    F{{to}} The value for the destination parameter used for WS-Addressing
-    F{{wsaAction}} The value for the action parameter used for WS-Addressing
-    F{{relatesTo}} The value for the relationship parameter used for WS-Addressing. In the form of a (URI, QName) pair
-    F{{relationshipType}} The value for the relationship type parameter used for WS-Addressing
-    F{{replyTo}} The value for the reply endpoint parameter used for WS-Addressing. This element must be present if a reply is expected. If this element is present, messageId must be present
-    F{{faultTo}} The value for the fault endpoint parameter used for WS-Addressing. If this element is present, messageId must be present
-    F{{messageId}} The value for the messageId parameter used for WS-Addressing
-    F{{username}} The value for the username parameter used for WS-Security Username Token
-    F{{password}} The value for the password parameter used for WS-Security Username Token
-    F{{passwordType}} The value for the password type parameter used for WS-Security Username Token
-}
+# Represents SOAP request.
+# + headers - The array of SOAP headers for the SOAP envelop to send to the endpoint
+# + payload - The xml of SOAP payload for the SOAP envelop to send to the endpoint
+# + soapVersion - The version of SOAP
+# + soapAction - The value of SOAP Action to send to the endpoint
+# + ^"from" - The value for the source endpoint parameter used for WS-Addressing
+# + to - The value for the destination parameter used for WS-Addressing
+# + wsaAction - The value for the action parameter used for WS-Addressing
+# + relatesTo - The value for the relationship parameter used for WS-Addressing. In the form of a (URI, QName) pair
+# + relationshipType - The value for the relationship type parameter used for WS-Addressing
+# + replyTo - The value for the reply endpoint parameter used for WS-Addressing. This element must be present if a reply
+# is expected. If this element is present, messageId must be present
+# + faultTo - The value for the fault endpoint parameter used for WS-Addressing. If this element is present, messageId
```

Comment:
We can update the formatting here as well.

Reply:
Fixed.